supfile.go

```diff
@@ -41,6 +41,7 @@ type Command struct {
 	Stdin  bool     `yaml:"stdin"`  // Attach localhost STDOUT to remote commands' STDIN?
 	Once   bool     `yaml:"once"`   // The command should be run "once" (on one host only).
 	Serial int      `yaml:"serial"` // Max number of clients processing a task in parallel.
+	RSync  []Upload `yaml:"rsync"`
```

Comment:
What I meant is that we should reuse `upload` yaml variable. I don't want to introduce a new one for the same purpose.